EXAMPLES.md

```diff
@@ -1,36 +1,36 @@
 # Example sites
 
-| Site                                                               |                   Source Code                            |
```

Comment:
can you revert the changes to that file? I would rather have a link yo your blog source in the docs when talking about dark mode example.

Reply:
Yes, no problem